implicit/evaluation.pyx

```diff
@@ -1,20 +1,19 @@
 # distutils: language = c++
 # cython: language_level=3
 
-from tqdm.auto import tqdm
-import numpy as np
-from scipy.sparse import coo_matrix, csr_matrix
 import cython
+import numpy as np
 from cython.operator import dereference
 from cython.parallel import parallel, prange
-from libc.stdlib cimport malloc, free
-from libc.string cimport memset
-from libc.math cimport fmin
+from scipy.sparse import coo_matrix, csr_matrix
+from tqdm.auto import tqdm
+
+from libc.stdlibcimportmalloc import free
```

Comment:
The build is failing because of some of these import changes - I think this will fix?

```suggestion
from libc.stdlib cimport malloc, free
from libcpp.unordered_set cimport unordered_set 
```